crates/username_attestation/coordinator/src/username_attestation.rs

```diff
@@ -75,6 +75,13 @@ pub fn get_all_username_attestations(_: ()) -> ExternResult<Vec<Record>> {
     query(filter)
 }
 
+
+/// Sign secret message with agent key as transferable proof that user owns this account
+#[hdk_extern]
+pub fn sign_message(message: SignableBytes) -> ExternResult<Signature> {
+    sign(agent_info()?.agent_initial_pubkey, message)
+}
+
```

Comment:
Why is signing now part of username_registry?

Reply:
as you can see the sign_message is two lines of code.. I did not feel it merited  a separate package.. and just added the function to wallet and username attestation zomes